app/src/main/java/com/teamagam/gimelgimel/app/model/entities/LocationEntityWithDB.java

```diff
@@ -0,0 +1,153 @@
+package com.teamagam.gimelgimel.app.model.entities;
+
+import android.content.AsyncQueryHandler;
+import android.content.ContentResolver;
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.net.Uri;
+
+import com.google.gson.annotations.SerializedName;
+import com.teamagam.gimelgimel.app.control.content_provider.GGContentProvider;
+import com.teamagam.gimelgimel.helpers_autodesk.control.content_provider.ProviderEntity;
+import com.teamagam.gimelgimel.helpers_autodesk.control.database.DbEntity;
+import com.teamagam.gimelgimel.helpers_autodesk.control.database.annotations.DbBinder;
+import com.teamagam.gimelgimel.helpers_autodesk.control.database.annotations.DbPrimaryKey;
+
+import java.io.Serializable;
+
+import static com.teamagam.gimelgimel.helpers_autodesk.control.content_provider.BaseContentProvider.formUri;
+
+/**
+ * This entity is a "friend" entity from GGMessagingAPI
+ * The naming convention is built to enable Gson instance to convert all fields from lower
+ * case with underscores to camel case and vice versa.
+ */
+//todo: update all columns and fields and api and...
+public class LocationEntityWithDB implements Serializable, DbEntity, ProviderEntity {
```

Comment:
What is this class?
If it's not used, it should be deleted. 
We have a source-control system if we will want to use it later
!fix